### DISCUSSION TITLE
what is difference between client.add and client.update?

### DISCUSSION THREAD

**Discussion by skim0011** [mem0ai/mem0#1968] [Q&A]
https://docs.mem0.ai/platform/quickstart#4-5-memory-history

client.add changes its info well, why do we need client.update? 

example:
**user alex info {name=alex, vegetarian=true}**
`message = "I recently tried chicken and I loved it. I'm thinking of trying more non-vegetarian dishes.."
client.add(memory_id, message, user_id="alex")`
**user alex info {name=alex, vegetarian=false}**

**Comment by Dev-Khant** [comment]
Hi @skim0011, you can use `client.add` to pass messages or text as input, which will create relevant memories automatically. To update a specific memory, use `client.update` with the memory ID and the new text you want to apply.
